streamlit_template/app.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import streamlit as st
+
+from toolkit.queries import (QUERY_ENTITY_DISTRIBUTION, QUERY_PROJECT_DOWNLOADS,
+                     QUERY_PROJECT_SIZES, QUERY_UNIQUE_USERS)
+from toolkit.utils import get_data_from_snowflake
+from toolkit.widgets import (plot_download_sizes, plot_unique_users_trend)
+
+
+# Custom CSS for styling
+with open('style.css') as f:
+    st.markdown(f'<style>{f.read()}</style>', unsafe_allow_html=True)
+
+def main():
+
+    # 1. Retrieve the data using your queries in queries.py
+    entity_distribution_df = get_data_from_snowflake(QUERY_ENTITY_DISTRIBUTION)
```

Comment:
I like and dislike that the app doesn't load until all the queries are done. Is there a giant spinning wheel we can wrap it in in the future?

Reply:
By default when I launch my example app I see this feature on the top right and the widgets load one at a time  (depending on which query finishes, and what data loads up first, I assume). I can launch the app after Sage Week so we can experiment with this.

<img width="632" alt="image" src="https://github.com/user-attachments/assets/74fa3c9b-5450-4a16-90ae-6eab71555222">